stdlib/errno.pyi

```diff
@@ -119,40 +86,135 @@ EHOSTUNREACH: int
 EALREADY: int
 EINPROGRESS: int
 ESTALE: int
-EUCLEAN: int
-ENOTNAM: int
-ENAVAIL: int
-EISNAM: int
-EREMOTEIO: int
 EDQUOT: int
 ECANCELED: int  # undocumented
-EKEYEXPIRED: int  # undocumented
-EKEYREJECTED: int  # undocumented
-EKEYREVOKED: int  # undocumented
-EMEDIUMTYPE: int  # undocumented
-ENOKEY: int  # undocumented
-ENOMEDIUM: int  # undocumented
 ENOTRECOVERABLE: int  # undocumented
 EOWNERDEAD: int  # undocumented
-ERFKILL: int  # undocumented
-EAUTH: int  # undocumented
-EBADARCH: int  # undocumented
-EBADEXEC: int  # undocumented
-EBADMACHO: int  # undocumented
-EBADRPC: int  # undocumented
-EDEVERR: int  # undocumented
-EFTYPE: int  # undocumented
-EL: int  # undocumented
-ELOCKUNMAPPED: int  # undocumented
-ENEEDAUTH: int  # undocumented
-ENOATTR: int  # undocumented
-ENOPOLICY: int  # undocumented
-ENOTACTIVE: int  # undocumented
-EPROCLIM: int  # undocumented
-EPROCUNAVAIL: int  # undocumented
-EPROGMISMATCH: int  # undocumented
-EPROGUNAVAIL: int  # undocumented
-EPWROFF: int  # undocumented
-EQFULL: int  # undocumented
-ERPCMISMATCH: int  # undocumented
-ESHLIBVERS: int  # undocumented
+
+if sys.platform != "win32":
+    ENOTBLK: int
+    EMULTIHOP: int
+    # All of the below are undocumented
+    EAUTH: int
+    EBADARCH: int
+    EBADEXEC: int
+    EBADMACHO: int
+    EBADRPC: int
+    EDEVERR: int
+    EFTYPE: int
+    ENEEDAUTH: int
+    ENOATTR: int
+    ENOPOLICY: int
+    EPROCLIM: int
+    EPROCUNAVAIL: int
+    EPROGMISMATCH: int
+    EPROGUNAVAIL: int
+    EPWROFF: int
+    ERPCMISMATCH: int
+    ESHLIBVERS: int
+
+    if sys.platform != "darwin" or sys.version_info >= (3, 11):
+        EQFULL: int  # undocumented
```

Comment:
Not possible to unflatten this nested check without triggering flake8-pyi's Y022